2_adventures/001/amaliahern/06/fizz_buzz.rb

```diff
@@ -0,0 +1,16 @@
+DIVISIBLE_BY_3 = "fizz"
+DIVISIBLE_BY_5 = "buzz"
+
+def fizzbuzz(number)
+  result = ""
+  if(number % 3 == 0)
+    result = DIVISIBLE_BY_3 
+  elsif(number % 5 == 0)
+    result << DIVISIBLE_BY_5 
+  elsif
+    result = number
+  end
+  result
```

Comment:
You can do this better without a branched conditional statement.  Tell your value to be shovelled on the result if some condition.  And do that as many times as you need to.

Then return what you want based on the result of the repetitive "filter".  Think about that toy that has the wholes that are smaller and smaller until finally, you have filtered everything but the answer.